modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/ddl/CreateTableCommand.java

```diff
@@ -273,30 +118,16 @@ public void ifNotExists(boolean ifNotExists) {
     }
 
     /**
-     * @return Data region name.
-     */
-    public String dataRegionName() {
-        return dataRegionName;
-    }
-
-    /**
-     * @param dataRegionName Data region name.
-     */
-    public void dataRegionName(String dataRegionName) {
-        this.dataRegionName = dataRegionName;
-    }
-
-    /**
-     * @return Encrypted flag.
+     * @return Primary key columns.
      */
-    public boolean encrypted() {
-        return encrypted;
+    public List<String> primaryKeyColumns() {
```

Comment:
Is it possible to make return types consistent in the next method's signatures?
`List<String> primaryKeyColumns` and `public Column[] columns() `

Reply:
got it !